config.json

```diff
@@ -3,7 +3,7 @@
   "maxTextSize": 1000000,
   "flowchart": {
     "useMaxWidth": false,
-    "htmlLabels": false
+    "htmlLabels": true
```

Comment:
This is the workaround for: https://github.com/mermaid-js/mermaid-cli/issues/266